plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/ideMaven/MavenRunnerUtils.kt

```diff
@@ -38,8 +38,20 @@ fun runMavenCopyCommands(sourceFolder: File, buildlogBuilder: StringBuilder, log
         val transformMvnRunner = TransformMavenRunner(project)
         val mvnSettings = MavenRunner.getInstance(project).settings.clone() // clone required to avoid editing user settings
 
+        // run copy dependencies
+        val copyDependenciesRunnable = runMavenCopyDependencies(sourceFolder, buildlogBuilder, mvnSettings, transformMvnRunner, destinationDir, logger)
+        copyDependenciesRunnable.await()
+        buildlogBuilder.appendLine(copyDependenciesRunnable.getOutput())
+        if (copyDependenciesRunnable.isComplete() == 0) {
+            val successMsg = "IntelliJ bundled Maven copy-dependencies executed successfully"
+            logger.info { successMsg }
+            buildlogBuilder.appendLine(successMsg)
+        } else {
+            emitMavenFailure("Maven Copy: bundled Maven failed: exitCode ${copyDependenciesRunnable.isComplete()}", logger)
```

Comment:
Outside of scope: 
Probably we want to emit this as a warning and not an error.